youtube_dl/YoutubeDL.py

```diff
@@ -833,6 +833,8 @@ def iter_playlistitems(format):
                                                       extra_info=extra)
                 playlist_results.append(entry_result)
             ie_result['entries'] = playlist_results
+            if result_type == 'playlist':
+                self.to_screen('[info] Playlist download process finished')
```

Comment:
This does not stylistically matches [opening message](https://github.com/hedii/youtube-dl/blob/output-message-when-playlist-downloaded/youtube_dl/YoutubeDL.py#L747), should be same prefix. Also it does not mention the playlist name.